packages/distance-sensor/src/i2c-utils.ts

```diff
@@ -0,0 +1,97 @@
+import {ExpansionHub} from "@rev-robotics/expansion-hub";
+
+export async function readRegister(
+    hub: ExpansionHub,
+    channel: number,
+    address: number,
+    register: number,
+): Promise<number> {
+    await hub.writeI2CSingleByte(channel, address, register);
+    await hub.readI2CSingleByte(channel, address);
+
+    return (await hub.getI2CReadStatus(channel)).bytes[0];
+}
+
+export async function readRegisterMultipleBytes(
+    hub: ExpansionHub,
+    channel: number,
+    address: number,
+    register: number,
+    n: number
+): Promise<number[]> {
+    await hub.writeI2CSingleByte(channel, address, register);
+    await hub.readI2CMultipleBytes(channel, address, n);
```

Comment:
`ExpansionHub.writeI2CReadMultipleBytes()` exists for this (though that's a terrible name, and its `startAddress` parameter name is also very unclear).